### ISSUE TITLE
Misleading error message from config parser

### ISSUE THREAD

**Issue by sur5r** [i3/i3status#249]
Given this (erroneous) configuration
```
general {
        output_format = "dzen2"
        colors = true
        interval = 5
}

order += "disk /home
order += "wireless wlan0"
order += "volume master"

wireless wlan0 {
        format_up = "W: (%quality at %essid, %bitrate) %ip"
        format_down = "W: down"
}

disk "/home" {
        format = "%free"
}

volume master {
        format = "♪: %volume"
        format_muted = "♪: muted (%volume)"
        device = "default"
        mixer = "Master"
        mixer_idx = 0
}
```
`i3status` reports:
```
bla.cfg:9: missing opening brace for section 'wireless'
```

While it's obvious how the parse reaches this conclusion, it's not all that helpful when searching for the actual mistake. Unfortunately, I'm not familiar with the parser code and have no time right now to look at it.

Maybe allowing new blocks to only start at the beginning of a line could improve this. While something like
``` 
order += "volume master" wireless wlan0 {
        format_up = "W: (%quality at %essid, %bitrate) %ip"
        format_down = "W: down"
}
```
seems to work just fine I doubt this is really needed.

**Comment by eplanet** [comment]
Hello and thanks for reporting the issue! I understand what you say, but it's not `i3status` code that produce that message since we rely on [libconfuse](https://github.com/martinh/libconfuse) to parse configuration using [cfg_parse](https://github.com/i3/i3status/blob/master/i3status.c#L552).

IMHO there's very little we can do without adding a lot of code, so I suppose you could report the error on the upstream library? That would be great!

**Comment by eplanet** [comment]
@sur5r any update on that?

**Comment by sur5r** [comment]
No. And I currently have no time to check back with upstream whether it's possible to improve this. Feel free to do so.

**Comment by eplanet** [comment]
After following up with `libconfuse`, one interesting fact is that strings are allowed to have multiple lines. In our case we assume strings to be one line only. Especially because the following configuration ouputs `something`, ignoring the actual beginning of the string:
```
order += "disk /"
disk "/" {
        format = "%free
something"
}
```

So I told we'd like them to implement a flag to force strings to be on one line only.
